### ISSUE TITLE
Possible courses

### ISSUE THREAD

**Issue by joel-becker** [joel-becker/research-ideas#1]
# MIT

## 14.386 New Econometric Methods

Lecture: T EVE (4-6 PM) (E51-372)

Focuses on recent developments in econometrics, especially structural estimation. Topics include nonseparable models, models of imperfect competition, auction models, duration models, and nonlinear panel data. Results illustrated with economic applications.

A. Abadie, W. Newey

## 14.382 Econometrics

Lecture: MW2.30-4 (E51-376) Recitation: F2.30-4 (E51-151) or F4-5.30 (E51-361) +final

Covers key models as well as identification and estimation methods used in modern econometrics. Presents modern ways to set up problems and do better estimation and inference than the current empirical practice. Introduces generalized method of moments and the method of M-estimators in addition to more modern versions of these methods dealing with important issues, such as weak identification or biases arising in high dimensions. Also discusses the bootstrap and explores very high dimensional formulations, or "big data." Students gain practical experience by applying the methods to real data sets. Enrollment limited.

W. Newey

## 14.160 Behavioral Economics

Lecture: MW10.30-12 (E52-432) Recitation: F10.30-12 (E51-372)

Covers recent theory and empirical evidence in behavioral economics. Topics include deviations from the neoclassical model in terms of (i) preferences (present bias, reference dependence, social preferences), (ii) beliefs (overconfidence, projection bias), and (iii) decision-making (cognition, attention, framing, persuasion), as well as (iv) market reactions to such deviations. Applications will cover a large range of fields, including labor and public economics, industrial organization, health economics, finance, and development economics.

A. Banerjee,  F. Schilbach

# Harvard

## Economics of Science

Thurs 3-6pm

Analyzes economic issues regarding the role of science and RD in the economy and in the deployment and productivity of scientists, engineers, and highly skilled technical workers. Topics include: wage levels/employment prospects; stipend policy, education/recruitment, student unionization/post-doc organization, career choices/trajectories, with reference to women; scientific competition/collaboration.

Richard Freeman

## Behavioural Dev Econ

Tues/Thurs 9:00am - 10:15am

This graduate level course will focus on the intersection of two rapidly growing fields in economics - development economics, and behavioral economics. We will study applications of behavioral economics to development questions, and ask whether there is a special behavioral science of poverty and development. Methods covered will include field experiments, lab experiments, tests of theory and combining experiments with structural estimation.

Gautam Rao

## Networks

Fri 3-6pm

This course prepares students for pure and applied research in the economic theory of networks. Topics covered include the macroeconomic network of production and trade; games of investment with social spillovers (education, criminal activity); financial contagion and other externalities; diffusion of beliefs and behaviors; segregation and homophily; and networked markets. The course emphasizes portable methods, including the linear algebra behind centrality measures; random graph theory; branching processes; power laws and Pareto tails. Prerequisites: basic microeconomic theory at the level of Econ 2010a,b; probability at the level of an advanced undergraduate course.

Ben Golub

## Econometric Methods

Tues/Thurs

This course continues the first year sequence in econometrics and covers a variety of topics and ideas that are important for pursuing and interpreting empirical research in economics. The first half of the course covers core econometric approaches that are important for a wide range of applications, including identification analysis, asymptotic approximations, large sample theory for estimation and hypothesis testing, and the bootstrap.  The second part of the course examines a range of complementary topics and new developments, including reasons why canonical econometric methods may be unreliable (such as model misspecification, identification failure, and the incidental parameters problem) and extensions of and alternatives to the traditional econometric paradigm (such as partial identification, Bayesian inference, nonparametrics, and machine learning).  Economic applications will be discussed throughout. Enrollment limited.

Andrews

## Foundations of Machine Learning

Thurs 3-6pm

The course provides an extensive account of the fundamental ideas underlying machine learning and the basic algorithms used in practice. The course first formalizes basic concepts used to establish the theory and language of machine learning. These concepts include PAC learnability, sample complexity, and the VC dimension. The course then covers the concepts of convexity, regularization, and stability as well as important algorithmic paradigms including stochastic gradient descent, boosting, support vector machines, kernel methods, feature selection, and neural networks.

Yaron Singer

**Comment by joel-becker** [comment]
## Discriminate between courses on same day

### Thurs 3-6pm

Foundations of ML, Economics of Science, New Econometrics Methods

### Fri 3-6pm

Networks

## Tues/Thurs 9-1030am

Behavioural dev

##
